manual/manual_03.asciidoc

```diff
@@ -1696,11 +1696,11 @@ limiting_attribute = attribute_definition
 
 
 If any of these limits are exceeded when trying to put (i.e. `locate`) anything inside the container, the statements in the corresponding `Else` part will be executed and the player turn aborted.
-In fact, these checks are performed because of the execution of a (((locating inside containers))) `Locate` statement (usually as a result of the player issuing a command with the intent of placing something in a container).
-This means that the execution of a sequence of statements can actually be interrupted in of the `Locate` statement by these limitations.
+In fact, these checks are performed because of the execution of a (((locating inside containers))) `Locate` statement (usually, but not limited to, as a result of the player issuing a command with the intent of placing something in a container).
+This means that a sequence of statements can actually be interrupted in the middle of its execution by a `Locate` statement triggering limitations.
```

Comment:
It is not "in the middle of the execution". It is exactly in that Locate statement, which might be anywhere. Maybe just remove the "in the middle of its execution" so it becomes "... be interrupted by a 'Locate' statement ..."